content/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Replication and rebalancing
+description: replication, rebalancing
+menu:
+    "5.1":
+        parent: Features
+        weight: 1
+---
+
+This page walks you through a simple demonstration of how TiKV replicates, distributes, and rebalances data. Starting with a 3-node local cluster, you'll: 
+
+1. Write some data via [go-ycsb](https://github.com/pingcap/go-ycsb) and verify that it replicates in triplicate by default.
+2. Add two more nodes and watch how TiKV automatically rebalances replicas to efficiently use all available capacity.
+
+{{< warning >}}
+Do not apply this in production.
+{{< /warning >}}
+
+
+## Prerequisites
+
+Make sure you have installed [TiUP](https://github.com/pingcap/tiup) version **v1.5.2** or above as described in [TiKV in 5 Minutes](../../tikv-in-5-minutes).
+
+
+## Step 1. Start a 3-node cluster
+
+Use the `tiup-playground` to start a 3-node local cluster.
+```sh
+tiup playground --mode tikv-slim --kv 3
+```
+
+The command above will output the following content:
+```txt
+...
+Starting component `playground`: /home/pingcap/.tiup/components/playground/v1.5.0/tiup-playground --mode tikv-slim --kv 3
+Using the version v5.0.2 for version constraint "".
+
+If you'd like to use a TiDB version other than v5.0.2, cancel and retry with the following arguments:
+    Specify version manually:   tiup playground <version>
+    Specify version range:      tiup playground ^5
+    The nightly version:        tiup playground nightly
+
+Playground Bootstrapping...
+Start pd instance
+Start tikv instance
+Start tikv instance
+Start tikv instance
+PD client endpoints: [127.0.0.1:2379]
+To view the Prometheus: http://127.0.0.1:33703
+To view the Grafana: http://127.0.0.1:3000
+```
+
+## Step 2. Write data
+
+On another terminal session, we will use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload.
+
+1. Clone the `go-ycsb` from GitHub.
+    ```sh
+    git clone https://github.com/pingcap/go-ycsb.git
+    ```
+2. Build the application from the source.
+    ```sh
+    make
+    ```
+3. Load a small workload using `go-ycsb`
+    ```sh
+    # By default, this workload will insert 1000 records into TiKV
+    ./bin/go-ycsb load tikv -P workloads/workloada -p tikv.pd="127.0.0.1:2379" -p tikv.type="raw" 
+    ```
+
+## Step 3. Verify replication
+
+1. To understand replication in TiKV, it's important to review a few concepts from the [architecture](https://github.com/tikv/tikv#tikv-software-stack).
```

Comment:
Since this is not a step, I suggest wrapping it by `info` block. BTW, you may want to link to [here](https://github.com/tikv/website/blob/1b03567b85756ad5a7bb7c40cc8174761bab8d09/content/docs/5.1/reference/architecture/overview.md) (if merged).

Reply:
Done.